ios/REANodesManager.m

```diff
@@ -52,6 +59,7 @@ @implementation REANodesManager
   NSMutableArray<REAOnAnimationCallback> *_onAnimationCallbacks;
   NSMutableArray<REANativeAnimationOp> *_operationsInBatch;
   REAEventHandler _eventHandler;
+  volatile void (^_mounting)(void);
```

Comment:
why `volatile`?

Reply:
That's because we use it frequently on two threads. I used volatile because I don't want threads to cache mounting.

Reply:
Why don't we use it with `shouldInterceptMountingBlock` too?

Reply:
Properties are atomic by default.